include/shared.h

```diff
@@ -192,7 +193,8 @@ extern int ft_socket_pair[2];
 extern int sock;
 extern int listen_sock;
 #define ADDR_OPTS "B:P:s:a:"
```

Comment:
It would be nice to have a 1:1 correspondence between XXX_OPTS and ft_xxx_usage().  E.g. ft_addr_usage() has opts B:P:s:, but ADDR_OPTS has B:P:s:a:.  Maybe that's the next step to this cleanup.